src/components/Buttons.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../components/Buttons.scss';
+
+const buttonCreator = (type) => {
+  const Button = ({ onClick, children, tabIndex, ...props }) => (
+    <button
+      className={type}
+      tabIndex={tabIndex}
+      {...props}
+      onClick={() => onClick()}
+    >
+      {children}
+    </button>);
+
+  Button.propTypes = {
+    onClick: PropTypes.func.isRequired,
+    children: PropTypes.node.isRequired,
+    tabIndex: PropTypes.string.isRequired,
```

Comment:
According to [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) the `tabIndex` is a `number`.

@banciur ... but we also made it `string` in [Link](https://github.com/Neufund/generic-ico/blob/master/src/components/Link.js#L22).